llvm/include/llvm/CodeGen/FastISel.h

```diff
@@ -275,7 +275,7 @@ class FastISel {
 
   /// This is a wrapper around getRegForValue that also takes care of
   /// truncating or sign-extending the given getelementptr index value.
-  Register getRegForGEPIndex(const Value *Idx);
+  Register getRegForGEPIndex(MVT PtrVT, const Value *Idx);
```

Comment:
The way to avoid the ABI break would be to keep the old `Register getRegForGEPIndex(const Value *Idx)` overload and make it pass `TLI.getPointerTy(DL)` for the first param to the other overload.